src/include/mpir_request.h

```diff
@@ -297,7 +297,7 @@ extern MPIR_Request MPIR_Request_direct[MPIR_REQUEST_PREALLOC];
 #ifdef MPICH_DEBUG_PROGRESS
 #define MPIR_REQUEST_SET_INFO(req, ...) \
     do { \
-        MPL_snprintf((req)->info, 100, __VA_ARGS__); \
+        MPL_snprintf_nowarn((req)->info, 100, __VA_ARGS__); \
```

Comment:
Just for background, we got rid of `MPL_snprintf` since it was added prior to C99 availability.